### PR TITLE
Task 3 on Bash realization 

### LINKED ISSUE #6: Task3_Bash
Create script that takes the file path and extension of files as arguments,
and returns count of files with the specified extension in this directory.

### PR DESCRIPTION
**Pull request by arkma**
## Describe your changes
Add task 3 realization on bash.

## Checklist before requesting a review.
- [x] I have performed a self-review of my code.
- [ ] My code follows the style guidelines of the project.
- [x] I have attached this pull request to a specific issue.
- [ ] I have added tests to validate the changes made.

## Additional Notes
Add any additional context, explanations, or concerns here.

### CODE REVIEW

**Review by OlehFedyshyn — APPROVED** [review]
All looks good

**Review by sbodia — APPROVED** [review]

File: task3.sh
Comment (AndriyGalias) [inline task3.sh:1]: Maybe it would be better put this logic into a function

File: task3.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+usage(){
+	cat << EOF
+Usage: $0 path ext
+path: Path to the directory
+ext: File extension
+EOF
+}
+
+if [ $# -ne 2 ]; then
```
Comment (AndriyGalias) [inline task3.sh:11]: here

File: task3.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+usage(){
+	cat << EOF
+Usage: $0 path ext
+path: Path to the directory
+ext: File extension
+EOF
+}
+
```
Comment (AndriyGalias) [inline task3.sh:10]: -------

File: task3.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# Get user parameters
+path="$1"
+ext="$2"
+
+if [ ! -d "$path" ]; then
+        echo "Path \"$path\" does not exist!"
+        exit 1
+fi
+
```
Comment (AndriyGalias) [inline task3.sh:25]: --------

**Review by AndriyGalias — APPROVED** [review]

**Review by Legamanchik — APPROVED** [review]

**Pull request merged**